src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
+$day-size: 100px;
+$day-border: 1px;
+$day-color: #eee;
+$day-color-hover: #ffbfcb;
+$day-font: Arial, sans-serif;
+$day-font-size: 30px;
+$gap: 1px;
+$columns: 7;
+$columns-size: $day-size + $day-border * 2;
+$max-width: $columns * $day-size + (($columns - 1) * $gap);
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: 0 auto;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: $max-width;
+
+  .calendar__day {
+    width: $day-size;
+    height: $day-size;
+    border: solid $day-border black;
+    background-color: $day-color;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    font-family: $day-font;
+    font-size: $day-font-size;
+    box-sizing: border-box;
+    cursor: pointer;
+    transition: linear 0.5s;
+
+    &:first-child {
+      margin-left: 606px;
```

Comment:
The `margin-left` value is hardcoded as `606px`. Consider calculating this value using the existing SCSS variables to maintain consistency and flexibility. For example, you could use `$columns-size * 6` or another appropriate calculation based on your layout requirements.